api/caseshare/index.ts

```diff
@@ -1,22 +1,23 @@
-import { Response } from 'express';
-import { getConfigValue } from '../configuration';
-import { SERVICES_CASE_SHARE_API_URL } from '../configuration/references';
-import { EnhancedRequest } from '../lib/models';
-import * as stubAPI from './stub-api';
+import { Response } from 'express'
+import { getConfigValue } from '../configuration'
+import { SERVICES_CASE_SHARE_API_URL } from '../configuration/references'
+import { EnhancedRequest } from '../lib/models'
+import * as stubAPI from './stub-api'
 
 // TODO: use below url for actual API
-const url: string = getConfigValue(SERVICES_CASE_SHARE_API_URL);
+// @ts-ignore
+const url: string = getConfigValue(SERVICES_CASE_SHARE_API_URL)
 
 /**
  * getRoot
  * example: /api/caseshare/
  */
 export async function getRoot(req: EnhancedRequest, res: Response) {
   if (process.env.STUB) {
```

Comment:
again please don't use ```process.env``` but instead abstract it out into a config variable, which can then be overridden by custom-environment-variables.json